settings/base.py

```diff
@@ -101,7 +101,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    #'django.middleware.clickjacking.XFrameOptionsMiddleware',
```

Comment:
@danaspiegel I commented this out because this middleware will prevent register site from being requested as an iframe source.